scio-core/src/main/scala/com/spotify/scio/coders/instances/AlgebirdCoders.scala

```diff
@@ -18,11 +18,10 @@
 package com.spotify.scio.coders.instances
 
 import com.spotify.scio.coders.Coder
-import com.twitter.algebird.{BF, Batched, CMS, TopK}
+import com.twitter.algebird.{Batched, CMS, TopK}
 
 trait AlgebirdCoders {
   implicit def cmsCoder[K]: Coder[CMS[K]] = Coder.kryo
-  implicit def bfCoder[K]: Coder[BF[K]] = Coder.kryo
```

Comment:
This might be a breaking change, since for Bloom Filters which are less than 10% filled, Algebird uses a Java EWAH Compressed Bitmap and we won't be having a Beam Coder for that. This means that if some one is using a Algebird Bloom Filter in their pipeline along with Scio, they would get an exception and coder derivation would fail. 

Should we keep this as is?

Reply:
it won't interfere. I'll revert this change.

Reply:
+1 in reverting it.